### ISSUE TITLE
Reflow comment shortcut (Ctrl + Shift + /) collides with new screen reader shortcut (also Ctrl + Shift + /)

### ISSUE THREAD

**Issue by DesiQuintans** [rstudio/rstudio#6189]
### System details

    RStudio Edition : Desktop
    RStudio Version :  1.3.797-2
    OS Version      :  Win-7 64-bit
    R Version       : 3.6.1

### Steps to reproduce the problem

Press `Ctrl + Shift + /` inside a long comment block. Instead of the comment being reflowed, it now brings up the _Confirm Toggle Screen Reader Support_ confirmation box.

### Describe the problem in detail

In previous RStudio versions, `Ctrl + Shift + /` was for reflowing comments. The new Preview has assigned the same shortcut to more than one editor function.

Furthermore, the screen reader shortcut cannot be modified in the _Modify Keyboard Shortcuts_ dialog.

### Describe the behavior you expected

The shortcut should have triggered the comment-reflow editor function, not the screen reader editor function.

**Comment by gtritchie** [comment]
Fixed in a8bbd93d

This is in daily builds, and will be in the next preview build.

(Changed screen reader shortcut to <kbd>Ctrl+Alt+Shift+/</kbd>)
